packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/input.overrides

```diff
@@ -42,36 +45,58 @@
 
 /* Icons */
 .ui.icon.input.small > input {
-    padding-right: calc(var(--scale-foxtrot) + @iconPadding) !important;
+    padding-right: var(--scale-golf)!important;
 }
+
 .ui.icon.input.large > input {
-    padding-right: calc(var(--scale-golf) + @iconPadding) !important;
+    padding-right: var(--scale-india) !important;
 }
 
 .ui.icon.input > svg.icon {
     top: 50%;
     transform: translateY(-50%);
     position: absolute;
-    right: var(--scale-bravo);
+    right: 10px;
```

Comment:
Make me wonder... shouldn't get these value from a calc based on our vars?

For example, if the padding is 40px which means `--scale-hotel` we could get the value from the following calc: `calc((var(--scale-hotel) / 2) / 2)`

Reply:
dans ce cas c'est plus icon size / 2 et non /2 /2 mais je te suis